src/Twig/IconExtension.php

```diff
@@ -72,6 +72,7 @@ final class IconExtension extends AbstractExtension
         'timesheet-team' => 'fas fa-user-clock',
         'plugin' => 'fas fa-plug',
         'configuration' => 'fas fa-cogs',
+        'paper-plane' => 'fas fa-paper-plane',
```

Comment:
Can we rename that to represent a functionality instead of simply shortening the name?
Maybe you can add a `mail` and `mail-sent` icon?

Reply:
So this one would be `mail-sent` obviously and `mail` = `fas fa-envelope-open`

Reply:
of course we can rename it, shall I update my pull request?

Reply:
Yes please 👍

Reply:
done